src/platforms/go/common/troubleshooting.mdx

```diff
@@ -19,3 +19,5 @@ supported:
 Since the Go SDK tries to read the source files from your local disk, it's possible that you’re compiling your binary and deploying it in a manner that makes the source code inaccessible. 
 
 This is common in Go because you can compile the binary, and then have no more need for the source code. Without access to the source code, however, we can’t map anything back, and we don’t support uploading Go source code with a release to stitch this data on.
+
+You can see [here](https://docs.sentry.io/platforms/go/serverless/#source-context) how to bunde the source code with the binary.
```

Comment:
```suggestion
You can see [in the Serverless section](serverless/#source-context) how to bundle the source code with the binary, which applies to other forms of deployment as well.
```

Just some ideas / suggestions:

- relative link will work during deployment preview and in production
- add note about the idea being applicable outside of Serverless (eventually we need to better organize the docs to avoid this potential for confusion)

Reply:
I agree with the suggestions from @rhcarvalho. I'd say apply those and you're good to go!